### PR TITLE
Standardize URL options, rename /zone_report to /zonereport, rename /access to /ticket

### PR DESCRIPTION
**Pull request by alanking**
All tests passed. Mostly find and replace

### CODE REVIEW

**Comment by alanking** [comment]
I can run clang-format on the renamed files if desired, but I didn't really touch anything beyond the strings that needed to be changed

**Comment by trel** [comment]
Yeah, we've got a couple more releases.    Your call.

**Comment by trel** [comment]
These look good to me - please # when ready.   Love the ticket work - so much cleaner/better.

File: packaging/irods_rest.py [hunk 1]
```diff
@@ -55,8 +55,14 @@ def logical_path_rename(_token, _src, _dst):
...
-         _recursive=None, _dst_resource=None,
+def logical_path_replicate(_token,
+                           _logical_path,
+                           _admin=None,
+                           _all=None,
+                           _num_threads=None,
+                           _repl_num=None,
+                           _recursive=None,
+                           _dst_resource=None,
          _src_resource=None):
```
Comment (alanking) [inline packaging/irods_rest.py:66]: Noooo

**Comment by trel** [comment]
yes, still a bit of inconsistency with src and dst and dest and such.  but we'll get there.

**Comment by alanking** [comment]
Agreed. We can make an issue for those once we've decided on what we want

**Pull request merged**